src/i22_bluesky/plans/stopflow.py

```diff
@@ -78,12 +85,32 @@ def stopflow(
     """
     stream_name = "main"
     flyer = HardwareTriggeredFlyable(StaticSeqTableTriggerLogic(panda.seq[1]))
-    devices = [flyer] + detectors + baseline
+    devices = [flyer] + detectors + [panda] + baseline 
+
+    # Collect metadata
+    plan_args = {
+        "pre_stop_frames": pre_stop_frames,
+        "post_stop_frames": post_stop_frames,
+        "exposure": exposure,
+        "shutter_time": shutter_time,
+        "panda": panda,
+        "detectors": [repr(device) for device in detectors],
+        "baseline": [repr(device) for device in baseline],
+    }
+    _md = {
+        "detectors": [device.name for device in detectors],
+        "plan_args": plan_args,
+        "hints": {},
+    }
+    _md.update(metadata or {})
+
+    # Add panda to collect from it
+    detectors = detectors + [panda]
```

Comment:
```suggestion
    # Add panda to collect from it
    detectors = detectors + [panda]
    
    _md = {
        "detectors": [device.name for device in detectors],
        "plan_args": plan_args,
        "hints": {},
    }
    _md.update(metadata or {})
```

PandA should be in `metadata["detectors"]` even though it's not in `metadata["plan_args"]["detectors"]`

Reply:
Done.